javascript/javascript2/week1/main.js

```diff
@@ -0,0 +1,51 @@
+//Find the shortest word
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function shortestWords(words) {
+  let shortWord = words[0];
+  for (const word of words) {
+    if (word.length < shortWord.length) {
+      shortWord = word;
+    }
+  }
+  return shortWord;
+}
+
+shortestWords(danishWords)
+
+console.log(shortestWords(danishWords));
+
+
+//Count danish word
+function getWord(words){
+  if(typeof words !== "string") {
+    return  "Please write something"
+  }
+  let specialLetter = ["å", "ø", "æ"]
```

Comment:
This can be const!
Variable that never changes should be const.

Reply:
Yes I will try to remember that, thanks